### PR TITLE
Update Prioritzed Backlog Issues' link

### PR DESCRIPTION
**Pull request by yoyoyojoe**
update url link for "Find an issue in Prioritized Backlog" to include "#column-16900748" so that it highlights the `Prioritized Backlog` column in `peopledepot`'s `Project`'s tab

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
 
 # PeopleDepot
 
-PeopleDepot is a project of Hack for LA, a brigade of Code for America. PeopleDepot provides backend infrastructure to provide a single source of truth on people, program areas, and project data for all Hack for LA projects. PeopleDepot uses a Django database and Django REST Framework that acts as a repository of information for other infrastructure projects (e.g., VRMS, Hack for LA, Civic Tech Index, Tables).
+PeopleDepot is a project of Hack for LA, a brigade of Code for America. PeopleDepot aims to provide a single source of truth as the backend infrastructure and database store for all Hack for LA projects, including data about people, program areas, and projects. PeopleDepot uses Django for its database and REST Framework and serves as a repository of information for other infrastructure projects (e.g., VRMS, Hack for LA Website, Civic Tech Index, Tables).
```
Comment (fyliu) [inline README.md:5]: The changes look good.

Not directly related to the current changes, 

1. We're only the data store for the organizational (or some word that means projects dealing with data that has to do with the internal infrastructure) projects for Hack for LA, not "all Hack for LA projects".

2. We're using PostgreSQL for the database, Django as the backend data model framework with Django REST Framework for the API layer.

File: README.md [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
 
-The hardest part of running an all volunteer open source project organization is managing the flow of information to all the people and projects that need it.  It can end up taking more time than the projects themselves. This project seeks to create an infrastructure that will automate everything that can be automated and do away with manual storage of duplicate information across projects.
+The hardest part about running a large organization using only free or open source tools and technologies is how to manage the flow of information and provide relevant info to all the people and projects that need it. Managing multiple databases inefficiently can end up taking more time than the projects themselves. This project seeks to create a maintainable database infrastructure that is synchronized. 
 
+In the process, it should allow for further automation and do away with manual storage of duplicate information across projects, which includes:
 - Recruiting members (Website: Project info and meeting times)
-- Onboarding members to resources (Tables:GitHub, Google Calendar, Google Drives, etc.)
+- Onboarding members to resources (Tables: GitHub, Google Calendar, Google Drives, etc.)
```
Comment (fyliu) [inline README.md:13]: tables and github are separate items

google tables

File: README.md [hunk 3]
```diff
@@ -23,16 +24,16 @@ The hardest part of running an all volunteer open source project organization is
...
 
 ## Contact info
 
-Contact us in the people-depot channel on Slack.
+Contact us in the `#people-depot` channel on Slack.
 
 ## Licensing
 
-Include details about the project's open source status.
+Open source licenses enable others to freely use, change, and distribute the project in your repository. For more information on repository licenses, see "[Licensing a repository][licensing-a-repo]." This repository uses the [GNU General Public License (v2.0)][licensing].
```
Comment (fyliu) [inline README.md:36]: we might even consider just using the last part to keep it simple.

File: README.md [hunk 4]
```diff
@@ -42,3 +43,5 @@ Include details about the project's open source status.
 [postgres-docs]: https://www.postgresql.org/docs/
 [contributing]: ./docs/contributing.md
 [HfLA onboarding]: https://www.hackforla.org/getting-started
+[licensing-a-repo]: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository
```
Comment (fyliu) [inline README.md:46]: cool! you know how to use this notation!

File: .github/ISSUE_TEMPLATE/blank-issue.md [hunk 5]
```diff
@@ -1,19 +1,23 @@
...
+about: Consistent formatting makes Issues concise and easy to navigate.
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
 ### Overview
-REPLACE THIS TEXT -Text here that clearly states the purpose of this issue in 2 sentences or less.
+Briefly describe the purpose of this issue.
```
Comment (fyliu) [inline .github/ISSUE_TEMPLATE/blank-issue.md:11]: hmm... the REPLACE... at the start of each line might be useful later on.

It allows a github action to auto reject an issue for not completing all the parts. these templates are copied from a set that hack for LA maintains and updates. I think it's fine to change their wording, just maybe have some common words to identify what still needs filling out.

**Review by fyliu — CHANGES_REQUESTED** [review]
Nothing major. I saw some things that was wrong from before. I can do a new PR later if you don't want to deal with them now.

File: README.md (hunk 1 above)
Comment (yoyoyojoe) [reply README.md:5]: I think it'd be good to improve the wording! Let me give it some thought.

File: README.md (hunk 2 above)
Comment (yoyoyojoe) [reply README.md:13]: I thought Tables was a project. Should we say Google Sheets? (Google Tables would sound funny.)

File: README.md (hunk 1 above)
Comment (yoyoyojoe) [reply README.md:5]: Taking your recommendation, what do you think about this?

> PeopleDepot is a project of Hack for LA, a brigade of Code for America. PeopleDepot aims to provide a single source of truth as the backend infrastructure and **_data store for Hack for LA projects_**, including data about people, program areas, and projects. **_PeopleDepot uses PostgreSQL for its database management and Django as the backend data model framework with Django REST Framework for the API layer. PeopleDepot's goal is to serve_** as a repository of information for other infrastructure projects (e.g., VRMS, Hack for LA Website, Civic Tech Index, Tables, etc).

File: README.md (hunk 3 above)
Comment (yoyoyojoe) [reply README.md:36]: Cool. A or B?

A.

> For more information on repository licenses, see "[Licensing a repository][licensing-a-repo]." This repository uses the [GNU General Public License (v2.0)][licensing].

B.

> This repository uses the [GNU General Public License (v2.0)][licensing].

File: .github/ISSUE_TEMPLATE/blank-issue.md (hunk 5 above)
Comment (yoyoyojoe) [reply .github/ISSUE_TEMPLATE/blank-issue.md:11]: Thanks for providing the context around this. 

> REPLACE THIS TEXT -Text here that clearly states the purpose of this issue in 2 sentences or less.

It looked tacky to me, but I can live with it if it serves a purpose. I looked at the `workflows` within `.github` but am only seeing one workflow, though (i.e., `new-issue-create-card.yml`)

In fact, I just created Test Issue #116 and I think I just reverse engineered creating an issue without following any of the issue templates... 😅

File: README.md (hunk 1 above)
Comment (fyliu) [reply README.md:5]: looks good

File: README.md (hunk 2 above)
Comment (yoyoyojoe) [reply README.md:13]: > I thought Tables was a project. Should we say Google Sheets? (Google Tables would sound funny.)

Never mind, Google Tables is a thing 👀

File: README.md (hunk 3 above)
Comment (fyliu) [reply README.md:36]: I think B is better since it's shorter

**Pull request merged**